content/en/blog/posts/2025-02-28-release-v0.15.md

```diff
@@ -0,0 +1,115 @@
+
+### Operator: Builds Bump
+
+The operator was updated to deploy Builds v0.15.1.
+
+## Installing Shipwright
+
+### Build
+
+1. Install Tekton v0.68.0:
+
+   ```bash
+   kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.68.0/release.yaml
+   ```
+
+2. Install v0.15.1 using the release YAML manifest:
+
+   ```bash
+   kubectl apply --filename https://github.com/shipwright-io/build/releases/download/v0.15.1/release.yaml --server-side
+
+   curl --silent --location https://raw.githubusercontent.com/shipwright-io/build/v0.15.1/hack/setup-webhook-cert.sh | bash
+   ```
+
+3. (Optionally) Install the sample build strategies using the YAML manifest:
+
+   ```bash
+   kubectl apply --filename https://github.com/shipwright-io/build/releases/download/v0.15.1/sample-strategies.yaml --server-side
+   ```
+
+### CLI
+
+#### Windows
+
+```sh
+curl --silent --fail --location https://github.com/shipwright-io/cli/releases/download/v0.15.0/cli_0.15.0_windows_x86_64.tar.gz | tar xzf - shp.exe
+shp version
+shp help
+```
+
+#### Mac
+
+```sh
+curl --silent --fail --location https://github.com/shipwright-io/cli/releases/download/v0.15.0/cli_0.15.0_macOS_$(uname -m).tar.gz | tar -xzf - -C /usr/local/bin shp
+shp version
+shp help
+```
+
+#### Linux
+
+```sh
+curl --silent --fail --location "https://github.com/shipwright-io/cli/releases/download/v0.15.0/cli_0.15.0_linux_$(uname -m | sed 's/aarch64/arm64/').tar.gz" | sudo tar -xzf - -C /usr/bin shp
+shp version
+shp help
+```
+
+### Operator
+
+To deploy and manage Shipwright Builds in your cluster, first ensure the operator v0.15.1 is installed and running on your cluster. You can follow the instructions on [OperatorHub](https://operatorhub.io/operator/shipwright-operator).
+
+Next, create the following:
+
+```yaml
+---
+apiVersion: operator.shipwright.io/v1alpha1
+kind: ShipwrightBuild
+metadata:
+  name: shipwright-operator
+spec:
+  targetNamespace: shipwright-build
+```
+
+
+## What About That .z?
+
+Since v0.14.0 was released, we have done a lot of work behind the scenes to automate Shipwright's
+release process and security posture. Part of this includes a set of nightly GitHub Actions that
+scan our container images for vulnerabilities at the code and operating system level. This process
+covers our most recent release as well as the nightly builds that come out of the `main` branch.
+
+Less than a day after Builds v0.15.0 was released, vulnerabilties in the `golang.org/x/crypto`
+and `golang.org/x/oath2` packages were disclosed. These were picked up by our nightly automation,
+which filed a GitHub issue notifying the community of the problem. The maintainers quickly sprung
+into action, submitting pull requests to patch the vulnerable code. The next night our automation
+detected these vulnerabilities were fixed, and drafted a security patch release.
+All this happened as the `cli` and `operator` projects were preparing releases of their own.
```

Comment:
Feel free to link to https://github.com/shipwright-io/build/blob/main/.github/workflows/report-release-vulnerabilities.yaml and https://github.com/shipwright-io/build/issues/1810 as examples.